.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+# command to install dependencies
+install:
+  - pip install -r requirements/travis_requirement.txt
```

Comment:
for now, let's just use the same requirements file...no need for a specific one for travis.